mediation-oauth/policy.bal

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2025 WSO2 LLC (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import choreo/mediation;
+
+TokenResponse? oauthAccessToken = ();
+http:Client? tokenClient = ();
+
+@mediation:RequestFlow
+public function oauthIn(mediation:Context ctx,
+        http:Request req,
+        string tokenEndpointUrl,
+        string clientId,
+        string clientSecret,
+        string headerName)
+                        returns http:Response|false|error? {
```

Comment:
```suggestion
        string headerName) returns http:Response|false|error? {
```

Reply:
This is a formatted code by the ballerina tool.